fastlane/metadata/default/release_notes.txt

```diff
@@ -1 +0,0 @@
-- Fixed a UI glitch in the authentication screen
```

Comment:
I'm pretty sure this is a byproduct of the code freeze starting from a commit, cf64f873, that had this and all the other localization files, but, when I finalized 2.11, there were no release notes translations available, so all these files –the translations for 2.10– were removed.

Ideally, they'll come back up-to-date once we'll finalize 2.12.